README.md

```diff
@@ -31,37 +25,28 @@ If you'd like to set up your own copy of the SIMPLE database, here's what we rec
 
 1. Clone or download a copy of this repo locally onto your computer.
  
-2. Set up an environment for the python code. 
+2. Set up an environment for the Python code and install dependencies. 
 A conda environment file `environment.yml` exists for convenience. The following commands will use that file to create and activate an 
    environment called `simple-db`:
 
     ```bash
     conda env create -f environment.yml
     conda activate simple-db
     ```
-    
-3. Install the latest version of the AstrodbKit2 package:
-    
-    ```bash
-     pip install git+https://github.com/dr-rodriguez/AstrodbKit2
-     ```
    
-4. Create an empty database and import the SIMPLE schema, and connect the database file `SIMPLE.db` as a Database object called `db
+3. In Python, connect a database file `SIMPLE.sqlite` as a Database object called `db` and recreate the database using the JSON files in the `data/` directory.
       
    ```python
-   from astrodbkit2.astrodb import create_database,Database
+   from astrodb_utils import load_astrodb
    from simple.schema import *
+   from simple.schema import REFERENCE_TABLES
    
-   connection_string = 'sqlite:///SIMPLE.db' # connection string for a SQLite database named SIMPLE.db
-   create_database(connection_string)
-   db = Database(connection_string)
-   ```
-5. Load in the database by reading in the directory where the JSON files are located
+   db = load_astrodb("SIMPLE.sqlite", recreatedb=True,  reference_tables=REFERENCE_TABLES)
+    ```
 
-   ```python
-   db.load_database('data/')
-   ```
-6. Use `astrodbkit2` to [explore](https://astrodbkit2.readthedocs.io/en/latest/#exploring-the-schema), [query](https://astrodbkit2.readthedocs.io/en/latest/#querying-the-database), and/or [modify](https://astrodbkit2.readthedocs.io/en/latest/#modifying-data) the database.
+    This step generates a "SIMPLE.sqlite" file which can be opened, explored, and modified using the [DB Browser for SQLite](https://sqlitebrowser.org/).
```

Comment:
Suggested rewrite:
`and modified using a variety of tools`
I know you love DB Browser, but I think the mention of it around line 13 is sufficient. I don't want to give the impression that this is the only/recommended way of using the database, **especially** for modifications.